src/pages/404.js

```diff
@@ -0,0 +1,31 @@
+import * as React from "react"
+
+import Container from "react-bootstrap/Container"
+import Row from "react-bootstrap/Row"
+import Col from "react-bootstrap/Col"
+
+import Layout from "../components/layout"
+import Seo from "../components/seo"
+
+const NotFoundPage = () => {
+  return (
+    <Layout>
+      <section className="bg-primary-subtle">
+        <Container className="py-5">
+          <Row>
+            <Col className="py-5">
+              <h1 className="mb-3 h2">404: Not Found</h1>
```

Comment:
Do we need to show `404` to the end user?

Reply:
We don't have to, but it's a good practice (especially in a blog) to keep users on the website.

The 404 page came with the boilerplate I reused as the base, thus, I decided to keep it

Reply:
or do you mean we should simply say `Page Not Found` without `404`?

Reply:
I mean to say Not Found without the HTTP code